js_modules/dagit/packages/core/src/ui/Button.tsx

```diff
@@ -0,0 +1,56 @@
+import {Button as BlueprintButton} from '@blueprintjs/core';
+import * as React from 'react';
+
+import {BaseButton} from './BaseButton';
+import {ColorsWIP} from './Colors';
+
+export const ButtonWIP = React.forwardRef(
+  (
+    props: React.ComponentProps<typeof BlueprintButton>,
+    ref: React.ForwardedRef<HTMLButtonElement>,
+  ) => {
+    const {children, intent, ...rest} = props;
+
+    const fillColor = React.useMemo(() => {
```

Comment:
I'm not sure the overhead of useMemo is worth it when this is a `{[intent: string]: string}` lookup?